### PR TITLE
Clean up license issues.

### PR DESCRIPTION
**Pull request by beorn7**
- Move CONTRIBUTORS.md to the more common AUTHORS.
- Added the required NOTICE file.
- Changed "Prometheus Team" to "The Prometheus Authors".
- Reverted the erroneous changes to the Apache License.

@juliusv

### CODE REVIEW

**Comment by brian-brazil** [comment]
:+1:

**Comment by beorn7** [comment]
@brian-brazil @discordianfish Please have a look at the NOTICE file. That would be a good place to mention Boxever or Docker as a company in the same style we mention SoundCloud there. Let me know if that's desired/required from your side.

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,19 @@
+The Prometheus project was started by Matt T. Proud in 2012.
```
Comment (juliusv) [inline AUTHORS]: I can haz fame? How about:

"The Prometheus project was started by Matt T. Proud and Julius Volz in 2012."

To be fair, that doesn't point out precisely that Matt had some very early code sketches (mostly client lib) before SC.

Or maybe:

"The Prometheus project was initially started (conceived?) by Matt T. Proud and then co-created together with Julius Volz in 2012."

Comment (juliusv) [inline AUTHORS]: Oh, and I forgot to mention that Matt explicitly told me he wanted us to add a notice here that he is now an alumni/emeritus, so he doesn't get a lot of mail/contacts about it. Maybe we should just put something like this behind his name: "Matt T. Proud (now emeritus)". Or something like that.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**